mechanics/TOC-REVIEW.schedule

```diff
@@ -43,4 +43,14 @@
 2021-12-02T08:30:00-08:00 | Eventing WG
 2021-12-09T08:30:00-08:00 | Networking WG
 2021-12-16T08:30:00-08:00 | DUX WG
-2021-12-23T08:30:00-08:00 | Security WG
+2022-12-23T08:30:00-08:00 | Security WG
```

Comment:
I think we skip this one. Could you please move all entries below by one week ? Thx!

Reply:
also `2022-12-23` does not look right either :-)

Reply:
D'oh -- you're totally right.

I need to make a script for this.